modules/claims_api/app/sidekiq/claims_api/poa_vbms_updater.rb

```diff
@@ -44,6 +44,11 @@ def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
       end
 
       poa_form.save
+    rescue BGS::ShareError => e
+      poa_form.status = ClaimsApi::PowerOfAttorney::ERRORED
+      poa_form.vbms_error_message = e.respond_to?(:message) ? e.message : 'BGS::ShareError'
+      poa_form.save
+      ClaimsApi::Logger.log('poa', poa_id: poa_form.id, detail: 'BGS Error', error: e)
```

Comment:
do we want a generic rescue as well? I think this works to catch that particular one but wondering if we need something kind of "fool proof". Can add as we improve v2 as well.